buildout.cfg

```diff
@@ -27,6 +27,9 @@ recipe = zc.recipe.egg
 eggs = PasteScript
        repoze.who
 
+[versions]
+Pygments = >=2.1
+
 [versions:python27]
 Sphinx = < 2
 webtest = < 3
```

Comment:
```suggestion
```

Reply:
I removed even more not needed version pins.